### DISCUSSION TITLE
How to add authentication for YouTube videos?

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [NickvisionApps/Parabolic#792] [Q&A]
I wanted to download a video from YouTube. But because of age restriction, it requires authentication. Now, how to add this. There are fields for username and password. What to write there?

![image](https://github.com/user-attachments/assets/25955ed2-9d44-4385-b709-7a01fee48816)
![image](https://github.com/user-attachments/assets/17225018-944e-4761-8359-b073146a831b)

**Comment by nlogozzo** [comment]
You don't need to specify a username and password. Instead you should go into the app's Preferences and provide a cookies file AFTER you have logged in to YouTube on the browser.
